### DISCUSSION TITLE
Elastic Agent support

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [MISP/PyMISP#797] [General]
Just wanted to check if SO is planning ELK SIEM support along with Elastic Agent in future.

**Comment by Rafiot** [comment]
I'm not sure anyone in the MISP team can answer this question (we don't use any of these tools), but PRs are welcome if someone feels like implementing something like that.

**Reply by Jackson-Pollock** [reply]
I'm sorry. Wrong channel.
